frontend/storybook/src/components/entry/homepage/footer/footer-jigs.ts

```diff
@@ -0,0 +1,30 @@
+import "@elements/entry/home/sections/footer-jigs";
+import "@elements/entry/home/TOSORT/column-list";
+
+export default {
+    title: 'Homepage Paragraph',
+  }
+  const STR_TITLE="JIGs";
+  const STR_LINE1="Hebrew JIGs";
```

Comment:
None of these seem to be used, please delete (or fill in their use, or add a comment explaining why they are left temporarily)